### DISCUSSION TITLE
Can I use Spatie media library with Fabricator?

### DISCUSSION THREAD

**Discussion by Xoshbin** [Z3d0X/filament-fabricator#78] [Q&A]
When I'm trying to add an image through Spatie media library I'm receiving this error:

> Call to undefined method Z3d0X\FilamentFabricator\Models\Page::getMedia()

I'm using Spatie media library for Filament
`                SpatieMediaLibraryFileUpload::make('image_url')->collection('pages'),
`

**Comment by Z3d0X** [comment]
You'll need to extend `Page` Model and configure it for Spatie Media Library.

PS: May i suggest an alternative: Filament Curator https://filamentphp.com/plugins/curator

**Comment by Xoshbin** [comment]
Thank you I will try
